apps/heureka/package.json

```diff
@@ -8,7 +8,11 @@
   ],
   "repository": "https://github.com/cloudoperators/juno/tree/main/apps/heureka",
   "license": "Apache-2.0",
-  "module": "build/index.js",
+  "exports": {
+    ".": "./build/index.js",
+    "./index.js": "./build/index.js",
+    "./package.json": "./package.json"
```

Comment:
why should we expose packge.json and index two times everywhere?

Reply:
Good point, it is a leftover from an attempt, but it is not necessary. I’ll remove it, thanks.